MaxText/configs/v5p/llama2_7b.sh

```diff
@@ -0,0 +1,50 @@
+# Llama2 7B model.
+# The batch size is set to achieve 4M token global batch size on 1 x v5p-512.
+# If running on a different slice size, modify the per_device_batch_size field
+# accordingly to achieve desired batch size.
+#
+# Command Flags:
+# OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
```

Comment:
Can you update the command flags requirement as well?

Thanks!

Reply:
Done